lib/oxidized/model/onefinity.rb

```diff
@@ -0,0 +1,19 @@
+class OneFinity < Oxidized::Model
+  using Refinements
+
+  # Fujitsu 1finity
+
+  prompt /(\r?[\w.@_()-]+[>]\s?)$/
+
+  cmd :all do |cfg|
+    cfg.each_line.to_a[1..-3].join
+  end
+
+  cmd 'show configuration | display set | nomore' do |cfg|
+    cfg
+  end
+
+  cfg :ssh do
+    pre_logout 'exit'
```

Comment:
Have you tried if `exec true` works for the model?
As all commands are independent, it would make the model independent from the prompt.
Note: `exec true` needs the ssh-server to support exec channels.

Reply:
Lines 14-16 would look like:
```ruby
cfg :ssh do
  pre_logout 'exit'
  exec true
end
```

Reply:
I added the line :)

I'm not sure I understand what it does because it's not in the docs but it didn't seem to break anything 👍